pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -1757,60 +1702,23 @@ func (s *AWSCloud) updateInstanceSecurityGroupsForLoadBalancer(lb *elb.LoadBalan
 		return fmt.Errorf("Could not determine security group for load balancer: %s", orEmpty(lb.LoadBalancerName))
 	}
 
-	// Get the actual list of groups that allow ingress from the load-balancer
+	// Gather all security groups
 	describeRequest := &ec2.DescribeSecurityGroupsInput{}
-	filters := []*ec2.Filter{}
-	filters = append(filters, newEc2Filter("ip-permission.group-id", loadBalancerSecurityGroupId))
-	describeRequest.Filters = s.addFilters(filters)
```

Comment:
I think it's risky to leave this around.  It seems wrong to change groups we don't have anything to do with, and it may not even be possible if people have locked down their IAM configuration.

I think maybe you could do a one-off fix (e.g. bash script) for your scenario, rather than embedding it into k8s?  (Particularly as I think you're not in a big group of people in this situation)